management-core/pom.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>management-app</artifactId>
+        <groupId>com.mx.task</groupId>
```

Comment:
You changed the package structure, but here you're referring the old structure

Reply:
Yes you are right now I know that its helpful delete the maven dependencies when I change names in packages, and maybe this was the cause from the compile  exceptions that you mention before